src/WattleScript.Interpreter/DataTypes/FieldInfo.cs

```diff
@@ -0,0 +1,18 @@
+﻿using WattleScript.Interpreter.Tree;
+
+namespace WattleScript.Interpreter
+{
+    internal class MemberFieldInfo
```

Comment:
File name matching class name. If we're using this to store function info as well, perhaps `WattleMemberInfo`?

Reply:
Resolved in ab2872c